src/api/core/componentStorage.ts

```diff
@@ -41,6 +42,13 @@ export interface IStorageComponent {
    */
   setDefault(auth: string): void;
 
+  /**
+   * clear / Remove a token with a specific key
+   *
+   * @param key
+   */
+  removeTokens(key: string): void;
```

Comment:
Is it Alias as a parameter? What does it mean key?

Reply:
Yes, will change it to `alias`! Good one!